docs/basics/descriptor.md

```diff
@@ -161,11 +161,17 @@ import reduxStore from './store';
 const MyElement = {
   name: {
     get: () => reduxStore.getState().name,
-    connect: (host, key, invalidate) => reduxStore.subscribe(invalidate),
+    connect: (host, key, invalidate) => reduxStore.subscribe(() => invalidate()),
```

Comment:
`reduxStore.subscribe(listener)` calls `listener` without arguments, to no change required here.